rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -4792,13 +4712,12 @@ bool opcode_019f(const ScriptArguments& args, const ScriptPlayer player, const S
 	opcode 01a0
 	@arg player 
 	@arg coord0
-	@arg coord1
+	@arg coord1 
 	@arg arg8 
 */
 bool opcode_01a0(const ScriptArguments& args, const ScriptPlayer player, const ScriptVec3 coord0, const ScriptVec3 coord1, const ScriptInt arg8) {
-	auto plyChar = player->getCharacter();
-	auto condition = plyChar->isStopped() && plyChar->getCurrentVehicle() != nullptr;
-	return script::objectInArea(args, plyChar, coord0, coord1, arg8, condition);
+	auto condition = player->getCharacter()->isStopped() && player->getCharacter()->getCurrentVehicle() != nullptr;
```

Comment:
probably makes more sense to first check the vehicle state.
The `isStopped` might not even be necessary and probably depends on knowing if the char or their vehicle has stopped. You should also check what the precise condition for a vehicle having stopped is (forward / backward speed close to zero? Overall speed close to zero? Characters speed close to zero? Handbrake applied?).

If we can't figure it out now, we should at least keep an eye on those details (and track them).

Reply:
As far I know function update will set flag stopped when speed is close to zero. I'll add extra condition checking vehicle.